src/key.js

```diff
@@ -518,14 +489,16 @@ function getExpirationTime(keyPacket, selfCertificate, defaultValue=null) {
  * - otherwise, returns the user with the latest self signature
  *
  * NOTE: call verifyPrimaryUser before calling this function.
+ * This is because getPrimaryUser isn't async, so it cannot validate and instead
```

Comment:
I didn't realize that. How should we detect that verifyPrimaryUser hasn't been called?

Reply:
When we discussed this in the channel I think Kay suggested setting the default of selfCertification.verified to null and then making sure it becomes false if verifyPrimaryUser is called. So that the null indicates that it was not verified.